lib/spack/llnl/util/filesystem.py

```diff
@@ -109,12 +110,17 @@ def path_contains_subdirectory(path, root):
     return norm_path.startswith(norm_root)
 
 
+def library_extensions():
```

Comment:
This should prob be a variable instead of a function

Reply:
Done